k8s.io/README.md

```diff
@@ -9,7 +9,7 @@ Vanity URL(s)
 |  | k8s.io | kubernetes.io |
 | --- | --- | --- |
 | APT downloads| https://apt.k8s.io | https://apt.kubernetes.io |
-| Blog | https://blog.k8s.io | |
```

Comment:
This is the wrong fix - we should fix the actual issue by making `https://blog.k8s.io/` go to the right place, i.e. update the nginx config in `k8s.io/configmap-nginx.yaml`.